wizeline/wize-q-remix
# feat: update login ui

#### What does this PR do?
- Updates Login UI

#### Where should the reviewer start?
- app/components/Login

#### How should this be manually tested?
1. Go to the Login page
2. Make sure that the component behaves normally and looks correctly in desktop and mobile version
3. 🎉 

#### What are the relevant tickets?
Closes #56 

#### Screenshots
<img width="1440" alt="Captura de Pantalla 2022-10-28 a la(s) 12 47 04" src="https://user-images.githubusercontent.com/71342528/198701216-f4761abb-2b19-44bd-8a2f-8c86a6ff9167.png">
<img width="216" alt="Captura de Pantalla 2022-10-28 a la(s) 12 47 30" src="https://user-images.githubusercontent.com/71342528/198701262-d4b497bf-2f6f-40c9-8178-91e7408367d8.png">
